Docs for v1 and v2

danielballan opened the pull request.

mrakitin commented on doc/source/v2/user/index.rst:
```diff
@@ -0,0 +1,275 @@
+   feedback.
+
+   For the stable usage "v1" usage, see :ref:`v1_index`. See
+   :ref:`transition_plan` for more information.
```
I understand it's a copy of the same block, but from `v2/index.rst`. I think it's OK to keep it twice for now, but we should de-duplicate it at some point.

danielballan replied:
Unfortunately, a limitation of rst is that you cannot include an rst file in another rst file. You can include a _txt_ file inside an rst file (which we do in just one place that I know of, whatsnew.rst) but I'm not sold that the extra ceremony is worth the de-duplication in this case.

mrakitin replied:
Agreed.

mrakitin commented on doc/source/v2/user/index.rst:
```diff
@@ -0,0 +1,275 @@
+--------------
+
+When databroker is first imported, it searches for Catalogs on your system,
+typically provided by a Python package or configuartion file that you or an
```
```suggestion
typically provided by a Python package or configuration file that you or an
```

mrakitin commented on doc/source/v2/user/index.rst:
```diff
@@ -0,0 +1,275 @@
+Look up a Run by ID
+-------------------
+
+Suppose know the unique ID of a run (a.k.a "scan") that we want to access. Note
```
```suggestion
Suppose you know the unique ID of a run (a.k.a "scan") that we want to access. Note
```

mrakitin commented on doc/source/v2/user/index.rst:
```diff
@@ -0,0 +1,275 @@
+See the documentation on dask_.
+
+TODO: This is displaying numpy arrays, not dask. Illustrating dask here might
+requires standing up a server.
```
```suggestion
require standing up a server.
```

mrakitin commented:
I saw a bunch of "Internal Server Errors":
```
ERROR:tornado.access:500 POST /v1/source (127.0.0.1) 5.06ms
```

Was the intake server actually started on Travis?

danielballan commented on doc/source/installation.rst:
```diff
@@ -0,0 +1,53 @@
+*We plan for facilities that rely on databroker to provide software
+distributions that bundle databroker itself and specific Catalogs for their
+users. We will list those distributions here. For users, this will be their
+one-stop stop; they need read no futher.*
```
Oops, should say "one-stop shot"

danielballan commented on doc/source/overview.rst:
```diff
@@ -0,0 +1,94 @@
+
+The :doc:`v1/index` documentation applies to ``databroker.v0`` and
+``databroker.v1``. The :doc:`v2/index` documentation applies to
+``databroker.v2``.
```
Thanks for the suggestion!

danielballan commented:
Good thought! I believe that if the server weren't started we should see connection errors. A `500` means that the client made a request that raised an unhandled Exception. In this case, I introduced a bug in 8c2ad66 that broke a critical code path, exercised by the server among other things. At the time I was focused on the docs rather than the tests or CI, so I didn't immediately notice it. I fixed the problem in a (much) later commit, 94e90ac.

danielballan commented:
Comments addressed. Thanks again for the long phone review and the careful read, @mrakitin. Very, very appreciated!

mrakitin commented on doc/source/installation.rst:
```diff
@@ -0,0 +1,53 @@
+*We plan for facilities that rely on databroker to provide software
+distributions that bundle databroker itself and specific Catalogs for their
+users. We will list those distributions here. For users, this will be their
+one-stop stop; they need read no futher.*
```
```suggestion
one-stop shop; they need read no further.*
```

mrakitin commented on doc/source/conf.py:
```diff
@@ -50,6 +50,7 @@
+    'matplotlib.sphinxext.plot_directive',
 ]
 
 extlinks = {'issue': ('https://github.com/NSLS-II/{0}/issues/%s'.format(repo_name),
```
```suggestion
extlinks = {'issue': ('https://github.com/bluesky/{0}/issues/%s'.format(repo_name),
```

mrakitin replied:
Not sure why the GH interface tries to touch the beginning of the line -- the line starts from the very beginning without blank spaces... Feel free to make the changes on your laptop instead.

mrakitin commented on doc/source/installation.rst:
```diff
@@ -0,0 +1,53 @@
+*We plan for facilities that rely on databroker to provide software
+distributions that bundle databroker itself and specific Catalogs for their
+users. We will list those distributions here. For users, this will be their
+one-stop stop; they need read no futher.*
```
```suggestion
one-stop shop; they need read no further.*
```

mrakitin replied:
For some reason it appeared twice, disregard this suggestion.
